lib/assets/javascripts/cartodb/table/menu_modules/filters/selection.js

```diff
@@ -293,9 +293,11 @@
     },
 
     _setupAutoComplete: function () {
+      var search_term;
       if(this.$el.find('.filterInputRemote').length > 0){
         this.$el.find('.filterInputRemote').autocomplete({
           source: function(request, response) {
+            search_term = request.term;
```

Comment:
It looks like you could define `var search_term =` here, right?

I've seen this pattern where you have to possible results in a conditional and you need to use it outside. i.e.:

```js
var foo;
if (condition) {
    foo = "bar";
} else {
    foo = "baz";
}

return foo;
```

But here it seems like you only need that variable inside the `if`. This is minor, of course, but still curious to what's your approach here!

Reply:
the variable is used in two locations.  300 and 323.  It is declared at line 296 so that it is accessible in both locations.